mozilla-services/socorro
# bug-1761322: add support for `Android_PackageName`

This adds support for the `Android_PackageName` annotation by adding it to the raw and processed crash schemas, adding it to super search fields, and adding it to the crash stats report view details tab.

This also updates the description and data review url for all the `Android_*` crash annotation fields in the raw crash schema.

To test:

1. run `make run`
2. process one or more Fenix crash reports
3. view a crash report
   1. verify that "Android Package Name" shows up in the report view
4. go to super search
   1. search for product=Fenix, android packagename=org.mozilla.fenix and verify that reports come up
   2. facet on android packagename and verify that the facet works
